crates/indexer/src/http/store_config.rs

```diff
@@ -54,8 +60,10 @@ struct SettingUri {
     creators: Option<Vec<Creator>>,
     #[serde(flatten)]
     extra: HashMap<String, serde_json::Value>,
+    auction_houses: Option<Vec<AuctionHouse>>,
 }
 
+#[allow(clippy::too_many_lines)]
```

Comment:
Please split this function up if possible rather than ignoring lints.

Reply:
@kespinola Need your guidance here.

Reply:
I'm going to keep this for now. The function isn't crazy big. Can look to refactor into helper functions in the future if we change the file again.